app/helpers/application_helper.rb

```diff
@@ -877,7 +877,7 @@ def model_from_active_tree(tree)
 
   def configuration_manager_scripts_tree(tree)
     case tree
-    when :automation_manager_cs_filter_tree
+    when :automation_manager_cs_filter_tree, :automation_manager_providers_tree
       "ManageIQ::Providers::AnsibleTower::AutomationManager::ConfiguredSystem"
     when :configuration_manager_cs_filter_tree
```

Comment:
I *think* Configuration > Management should have the same issue, missing `:configuration_manager_providers_tree` here, still checking..

Reply:
Never mind, Foreman seems to work even without that change.